src/Sylius/Component/Core/Model/ProductInterface.php

```diff
@@ -47,12 +47,12 @@
     /**
      * @return string
      */
-    public function getSku();
+    public function getCode();
 
     /**
-     * @param string $sku
+     * @param string $code
      */
-    public function setSku($sku);
+    public function setCode($code);
```

Comment:
It does [here](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Product/Model/ProductInterface.php#L33)